### PR TITLE
Feat/table menu

### PR DESCRIPTION
**Pull request by AdamSakeA**

### CODE REVIEW

File: client/src/pages/Products/components/DeletedHistory/index.jsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+              <th>
+                <FormattedMessage id="app_table_type" />
+              </th>
+              <th>
+                <FormattedMessage id="app_table_action" />
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            {products.map((product, index) => (
```
Comment (radenrinoto) [inline client/src/pages/Products/components/DeletedHistory/index.jsx]: add conditional chaining method

File: client/src/languages/en.js
Comment (radenrinoto) [inline client/src/languages/en.js:1]: update to en.js in backend

File: client/src/languages/id.js
Comment (radenrinoto) [inline client/src/languages/id.js:1]: update to id,js in backend

File: client/src/pages/Products/components/DeletedHistory/index.jsx (hunk 1 above)
Comment (AdamSakeA) [reply client/src/pages/Products/components/DeletedHistory/index.jsx]: allright done mas rhino thnk you

**Pull request merged**